src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -24,7 +24,7 @@ public FunctionsHostingConfigOptions()
         /// <summary>
         /// Gets a value indicating whether worker concurrency feature is enabled in the hosting config.
         /// </summary>
-        public bool FunctionsWorkerDynamicConcurrencyEnabled
+        internal bool FunctionsWorkerDynamicConcurrencyEnabled
```

Comment:
Are you changing visibility to avoid having this being evaluated/set by `Bind()` calls in case that ever happens again? With the current changes, this would no longer be a requirement, right?

Reply:
Correct. But it's a double-protection. Because the `Bind()` wasn't obvious previously, I was worried someone could inadvertently add some code that calls `Bind()` again. There's no reason for these to be public, and it makes it impossible for them to be set by any future `Bind()`.